### PR TITLE
 Edit  index, image and css file to include media for desktop version

### PR DESCRIPTION
**Pull request by dfizzkruah93**
Update ### Update for Desktop version

In this project:

- Added media queries for the desktop version
- Include menu for display
- Update the middle section for the desktop version
- Update index file to for desktop version
- Change the background image
- Add new images
- Add style for navigation element

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,305 +1,295 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <title>Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./css/style.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+    <header class="navbar">
```
Comment (BenMukebo) [inline index.html:12]: - [ ] Don't you think like me? adjusting your padding-top for the header can make it more pretty, the space shown is too large, please reduce it a bit.
- [ ] 
headersec

File: index.html [hunk 2]
```diff
@@ -1,305 +1,295 @@
...
+                        <a href="#"><img src="./img/hn.svg" alt="HN"></a>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="works">
+            <div class="single-work">
+                <img src="./img/Portfolio6.png" alt="Portfolio 1">
+                <div class="holder">
+                    <h2>Tonic</h2>
```
Comment (BenMukebo) [inline index.html:73]: - [ ] Your design is awesome, great job... but there is a small issue in the positioning of your cards, they are reversed 😄 , the image should be on the left and the text on the other side as per the Figma design, please fix that, 
- [ ] ⚠️ `not only this card, but all of them have to be arranged, please take a look back to this` [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848)

do this

File: index.html [hunk 3]
```diff
@@ -1,305 +1,295 @@
...
+            <div class="form">
+                <form action="https://formspree.io/f/xoqrdelo" method="post">
+                    <ul>
+                        <li><input type="text" name="name" placeholder="Your Name" maxlength="30" required></li>
+                        <li><input type="email" name="email" placeholder="Your Email" required></li>
+                        <li><textarea name="message" placeholder="Write your message here" maxlength="500"
+                                required></textarea></li>
+                        <li><button class="button-submit" type="submit">Get It Touch</button></li>
+                    </ul>
+                </form>
```
Comment (BenMukebo) [inline index.html:284]: - [ ] Please add some width to the form and some few 🤌  `padding` to separate the title from the `p` and `p` from the form, they are so close and that is not fine for the user readability

- [ ] YOUR DESIGN  
contsc

- [ ] THE FIGMA
GIF

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi ** @dfizzkruah93, **

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (dfizzkruah93) [reply index.html:12]: @BenMukebo 

Thanks for your feedback. I'm working on it

File: index.html (hunk 3 above)
Comment (dfizzkruah93) [reply index.html:284]: @BenMukebo . Please review changes that I have made and additional commits

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @dfizzkruah93, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dfizzkruah93** [comment]
Thanks for the review

**Pull request merged**